package.json

```diff
@@ -4,7 +4,7 @@
   "description": "AsyncAPI website",
   "scripts": {
     "dev": "node scripts/index.js && next -p 3000",
-    "build": "node scripts/index.js && next build",
+    "build": "node scripts/build-tsc-members.js && node scripts/index.js && next build",
```

Comment:
you can add the `build-tsc-memebers.js` in scripts/index.js` like all of the other scripts so we don't have to clutter the package.json file.